collect_app/src/main/java/org/odk/collect/android/audio/AudioHelper.java

```diff
@@ -31,21 +33,26 @@
  * construct multiple instances (within a {@link android.view.View} or
  * {@link androidx.fragment.app.Fragment} for instance) if needed within one
  * {@link android.app.Activity}.
+ *
+ * @deprecated wrapping the ViewModel like this doesn't really fit with other ways we've integrated
+ * widgets with "external" services. Instead of this widgets should talk to {@link AudioPlayer}
+ * and the Activity/Fragment components should talk to the ViewModel itself.
  */
 
+@Deprecated
```

Comment:
I definitely could have removed this everywhere but it felt like it would be pretty time intensive and actually pretty isolated from what I was working on. We can come back for this next time we touch anything in the neighbourhood.